sycl/doc/extensions/IntelGPU/IntelGPUDeviceInfo.md

```diff
@@ -9,9 +9,50 @@ This proposal details what is required to provide this information as a SYCL ext
 
 ## Feature Test Macro ##
 
-The Feature Test Macro will be defined as:
+The Feature Test Macro SYCL\_EXT\_INTEL\_DEVICE\_INFO will be defined as one of the values defined in the table below. The existence of this macro can be tested to determine if the implementation supports this feature, or applications can test the macro's value to determine which of the extension's APIs the implementation supports.
+
+| Value | Description |
+| ----- | ----------- |
+| 1     | Initial extension version\. Base features are supported |
+| 2     | Device UUID is supported |
+
+
+# Device UUID #
+
+A new device descriptor will be added which will provide the device Universal Unique ID (UUID).
+
+This new device descriptor is currently only available for devices in the Level Zero platform, and the matching aspect is only true for those devices. The DPC++ default behavior would be to expose the UUIDs of all supported GPU devices which enables detection of total number of unique devices.
```

Comment:
Could you please clarify if there any reason to say "GPU" here? Won't this feature work for other device types?

Reply:
AFAIK, there is no restriction that it won't work for other devices. I'll remove it. thanks.

Reply:
I thought this query is going to be implemented only for the Level Zero backend, which works only for GPU devices.  Are we planning to implement it on other backends too?

Reply:
this will be on opencl as well.